Fixed part 2 of issue3080: UI suspension button for CronJob

EdanSneh opened the pull request:
Creates new button for suspending cronjob, with a component and a controller. New suspend API handler in backend, with a suspension library.

EdanSneh commented:
/assign @maciaszczykm

maciaszczykm commented on src/app/frontend/common/components/resourcecard/resourcecardsuspendmenuitem_component.js:
```diff
@@ -0,0 +1,121 @@
+    this.state_.reload();
+  }
+
+  successenable_() {
```
Use camelCase.

maciaszczykm commented on src/app/frontend/common/components/resourcecard/resourcecardsuspendmenuitem_component.js:
```diff
@@ -0,0 +1,121 @@
+        });
+  }
+
+  successdisable_() {
```
Use camelCase.

maciaszczykm commented on src/app/backend/suspend/suspender.go:
```diff
@@ -0,0 +1,49 @@
+  }
+
+  if suspend == "true" {
+  *cron.Spec.Suspend = true
```
Run code formatting.

EdanSneh commented:
@maciaszczykm I implemented the changes you requested. Not sure why travis is failing now
@austbot I added specs, but I still don't think it will fulfill codecov

maciaszczykm commented on src/app/backend/suspend/suspender.go:
```diff
@@ -0,0 +1,49 @@
+		return nil, err
+	}
+
+	if suspend == "true" {
```
Why suspend is not a boolean?

maciaszczykm commented:
I am closing it as we have just finished migration and it is stale. Please reopen if you would like to finish it.

/close
